tensorflow/lite/micro/micro_graph.h

```diff
@@ -60,12 +60,15 @@ class MicroGraph {
   // Get the specified input tensor of a specified subgraph in the model.
   virtual TfLiteEvalTensor* GetSubgraphInput(int subgraph_idx, int input_idx);
 
-  // Number of tensor outputs to a specified subgraph in the model.
+  // Number of tensor outputs from a specified subgraph in the model.
   virtual size_t NumSubgraphOutputs(int subgraph_idx);
 
   // Get the specified output tensor of a specified subgraph in the model.
   virtual TfLiteEvalTensor* GetSubgraphOutput(int subgraph_idx, int output_idx);
 
+  // Number of operators in the specified subgraph in the model.
+  virtual int NumSubgraphOperators(int subgraph_idx);
```

Comment:
I think I got confused in the internal review of this change. If this is a new function added in the current change, I think it is not needed anymore (now that we have the free functions in flatbuffer_utils.cc).

Reply:
I made a mess by merging the origin branch before pushing my update. It overwrote my changes.

Reply:
OK, I think I fixed it.